yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/AnchorConverterTest.java

```diff
@@ -39,6 +39,7 @@
     private static final String PASSPORT_SOURCE_TYPE = "PASSPORT";
     private static final String DRIVING_LICENCE_SOURCE_TYPE = "DRIVING_LICENCE";
     private static final String YOTI_ADMIN_VERIFIER_TYPE = "YOTI_ADMIN";
+    private static final String YOTI_UNKNOWN_ISSUER = "CN=document-registration-server";
```

Comment:
Could this be in `TestAnchors.java`?

Reply:
Yep it absolutely could do - I have put it in the test file to follow convention of the other constants.  We could rename it to just UNKNOWN_ISSUER to follow the convention used.

Reply:
Ah yes, I see what you mean - maybe it should be `DOCUMENT_REGISTRATION_ISSUER` and remain where it is - @echarrod thoughts?

Reply:
I think naming it `DOCUMENT_REGISTRATION_ISSUER` and leaving it where it is seems sensible.